tests/01-image.test.sh

```diff
@@ -28,8 +28,33 @@ function run_test {
   fi
   popd
 
+  ttest "Dbless Test"
+  
+  pushd compose
+  docker-compose up -d
+  until docker ps -f health=healthy | grep -q ${KONG_DOCKER_TAG}; do
+    docker-compose up -d
+    docker ps
+    sleep 15
+  done
+  
+  curl -I localhost:8001 | grep -E '(openresty|kong)'
+  if [ $? -eq 0 ]; then
+    tsuccess
+  else
+    tfailure
+  fi
+  
+  docker-compose kill
+  docker-compose rm -f
+  sleep 5
```

Comment:
I pressume there isn't something we can actively wait for instead of this.

Reply:
whoops I missed this comment.

Not easily no (or at least that I know of).

Reply:
Maybe along the lines of https://github.com/Kong/kong-build-tools/blob/master/test/util.sh#L45?